src/components/tile-select/tile-select.pw.tsx

```diff
@@ -0,0 +1,610 @@
+
+        await expect(inputElement(page)).toHaveCSS("border-color", borderColor);
+      });
+    });
+
+    ["1", "2", "3", "4"].forEach((value) => {
+      test(`should check when value prop is set as ${value}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<TileSelectComponent value={value} />);
+
+        await expect(inputElement(page)).toHaveAttribute("value", value);
+      });
+    });
+
+    testData.forEach((id) => {
+      test(`should check id set as ${id}`, async ({ mount, page }) => {
+        await mount(<TileSelectComponent id={id} />);
+
+        await expect(inputElement(page)).toHaveAttribute("id", id);
+      });
+    });
+
+    testData.forEach((name) => {
+      test(`should check name set as ${name}`, async ({ mount, page }) => {
+        await mount(<TileSelect name={name} />);
+
+        await expect(inputElement(page)).toHaveAttribute("name", name);
+      });
+    });
+
+    test(`should check when tile is selected`, async ({ mount, page }) => {
+      await mount(<TileSelect checked />);
+
+      await expect(inputElement(page)).toHaveAttribute("aria-checked", "true");
+    });
+
+    test("should check when tile is not selected", async ({ mount, page }) => {
+      await mount(<TileSelect checked={false} />);
+
+      await expect(inputElement(page)).toHaveAttribute("aria-checked", "false");
+    });
+
+    test(`should render TileSelect component with className`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TileSelectComponent className="tile-select-classname" />);
+
+      await containsClass(
+        tileSelectDataComponent(page),
+        "tile-select-classname"
+      );
+    });
+
+    test(`should check footer prop`, async ({ mount, page }) => {
+      await mount(<WithAFooter />);
+
+      const buttonElement = getComponent(page, "button");
+      await buttonElement.isVisible();
+      await expect(buttonElement).toHaveText("Footer Button");
+    });
+
+    test(`should check prefixAdornment prop`, async ({ mount, page }) => {
+      await mount(<PrefixAdornmentComponent />);
+
+      const pill1 = getComponent(page, "pill").nth(0);
+      const pill2 = getComponent(page, "pill").nth(1);
+      await expect(pill1).toHaveText(CHARACTERS.DIACRITICS);
+      await expect(pill1).toBeVisible();
+      await expect(pill2).toHaveText(CHARACTERS.SPECIALCHARACTERS);
+      await expect(pill2).toBeVisible();
+    });
+
+    test(`should check additionalInformation prop`, async ({ mount, page }) => {
+      await mount(<WithAdditionalInformation />);
+
+      const pill1 = getComponent(page, "pill").nth(0);
+      await expect(pill1).toHaveText("Message");
+      await expect(pill1).toBeVisible();
+    });
+
+    test(`should check when accordionExpanded set as false`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<AccordionTileSelectComponent accordionExpanded={false} />);
+
+      const getElementInput = getElement(
```

Comment:
**suggestion(non-blocking):** this could be given a better variable name. Something like `tileSelectAccordionChildren` is a bit more descriptive.